config/database.yml

```diff
@@ -58,6 +58,10 @@ development:
 test:
   <<: *default
   database: newsletterApplicaton_test
+  host: localhost
+  port: 5432
+  username: rail_user
+  password: Redhat!123
```

Comment:
I think we need to keep the credentials hidden, can you also remove this file and add it to .gitignore?

In foreman we created the same file with .example in the end of file name so devs could reuse it as a template.

Reply:
@Ron-Lavi can we do in this way, with `EDITOR=nano rails credentials:edit` , update username and password here. Then update the database.yml with:

```
username: <%= Rails.application.credentials.database[:username] %>
password: <%= Rails.application.credentials.database[:password] %>
```

Reference: https://levelup.gitconnected.com/setting-up-postgres-with-ruby-on-rails-7-on-ubuntu-20-04-45ffd75e863d

Reply:
Actually not a bad idea, let's try it maybe in a follow up, for now I suggested using it as in Foreman